support/mem/intune/device-protection/troubleshoot-conditional-access.md

```diff
@@ -59,6 +59,8 @@ You can view these conditions for each device in the Azure portal and in the dev
   3. Select the *Enable Browser Access* button.
   4. In the Chrome browser, sign out of Microsoft 365 and restart Chrome.
 
+Desktop applications must use modern authentication mechanisms, which rely on users interacting with the identity provider using a web browser. Legacy authentication mechanisms such as apps [sending the password directly](https://learn.microsoft.com/azure/active-directory/develop/v2-oauth-ropc) are not supported.
+- 
```

Comment:
Do you think we need add clarity on which platform it is supported and where it is not, and in which configuration:

1. Desktop applications - must user mordern authentication.
2. Script/automation - can user MSAL.NET with Broker on Win10+.